torch_xla/csrc/ops/threshold.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "ir.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+
+// IR node for the threshold operation.
```

Comment:
We can avoid those comments ;)
You can leave it, but in general they do not add much.